docs/reference/deprecation-policy.md

```diff
@@ -70,97 +70,196 @@ API version at least as stable is released.**
 GA API versions can replace GA API versions as well as beta and alpha API
 versions.  Beta API versions *may not* replace GA API versions.
 
-**Rule #4: Other than the most recent API versions in each track, older API
+**Rule #4a: Other than the most recent API versions in each track, older API
 versions must be supported after their announced deprecation for a duration of
 no less than:**
 
    * **GA: 1 year or 2 releases (whichever is longer)**
-   * **Beta: 3 months or 1 release (whichever is longer)**
+   * **Beta: 6 months or 2 releases (whichever is longer)**
    * **Alpha: 0 releases**
 
-This is best illustrated by example.  Imagine a Kubernetes release, version X,
-which supports a particular API group.  A new Kubernetes release is made every
-approximately 3 months (4 per year).  The following table describes which API
-versions are supported in a series of subsequent releases.
+NOTE: Until [#52185](https://github.com/kubernetes/kubernetes/issues/52185) is
+resolved, no API versions may be removed.
+
+**Rule #4b: The "preferred" API version and the "storage version" for a given
+group may not advance util after a release has been made that supports both the
+new version and the previous version**
+
+Users must be able to upgrade to a new version of Kubernetes and then roll back
+to a previous version, without converting anything to the new API version or
```

Comment:
to a previous release.

Elsewhere in this doc we use "release" for discussing releases and "version" with respect to API versions.